focuspoints.lrdevplugin/DefaultPointRenderer.lua

```diff
@@ -164,3 +201,36 @@ function DefaultPointRenderer.createPointView(x, y, rotation, iconFileTemplate,
 
   return view
 end
+
+--[[
+-- Takes a LrPhoto and returns the rotation and horizontal mirroring that the use has set in Lightroom (generaly in grid mode)
+-- photo - the LrPhoto to calculate the values from
+-- returns:
+-- - rotation in rad in trigonometric sense
+-- - horizontal mirroring (0 -> none, -1 -> yes)
+--]]
+function DefaultPointRenderer.getLightroomRotationAndMirroring(photo)
+  local lrRotation = photo:getRawMetadata("orientation")
```

Comment:
yes totally undocumented, unbelievable really. I stumbled it on the sdk forum in a post which is interesting for some other informations by the way:
https://forums.adobe.com/thread/1822776